src/main/webapp/resources/js/pages/analysis/components/Analysis.jsx

```diff
@@ -52,6 +52,13 @@ const analysisErrorTabKeys = [
   "standard-error",
   "standard-out"
 ];
+const analysisSistrTabKeys = [
```

Comment:
These are specific for sistr and therefore should be on a sistr component?  

As well, you have an array `analysisTypesWithAdditionalPage` that is not used.

Reply:
These are used to update the "nav". This is all handled by Analysis.jsx that's the reason for it being there. Should be able to move this into the sistr component but then wouldn't we need to use a context to access it?